source/_includes/puppet-collections/_gpg_command_output.md

```diff
@@ -0,0 +1,2 @@
+    gpg: Signature made Mon Nov 30 10:47:39 2015 PST using RSA key ID 4BD6EC30
+    gpg: Good signature from "Puppet Labs Release Key (Puppet Labs Release Key) <info@puppetlabs.com>"
```

Comment:
Since the new key isn't live yet, I need updated output for this. I can update the key ID manually and guess at the rest, but if someone can generate the correct output, that would be preferable.